.github/workflows/CD.yml

```diff
@@ -21,11 +22,6 @@ jobs:
       - uses: actions/checkout@v4
         with:
           persist-credentials: false
-      - name: Install toolchain
-        if: ${{ runner.os == 'Windows' }}
-        run: |
-          choco install rtools -y --no-progress --force --version=4.0.0.20220206
-          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
```

Comment:
Default toolchain installed on Windows runners is fine, but it needs a bit more work to get right (we need to repair the wheel now to bring in DLLs the user's machine will not have, but didn't before. Why now? No clue).

In any case, this bumps our tools quite a bit - from GCC 10.3 to 14.2 now. That fixes the issue we had with Meson in #955.